util/include/digest.hpp

```diff
@@ -18,9 +18,7 @@
 
 namespace concord::util::digest {
 
-using BlockDigest = std::array<std::uint8_t, DIGEST_SIZE>;
-
-#if defined USE_CRYPTOPP_HASH
+#if defined USE_CRYPTOPP_SHA_256
```

Comment:
#ifdef should include appropriate includes as well

Reply:
Included in header inclusion.

Reply:
This means that it shouldn't be in this file, only in a single header.